src/main/java/net/minestom/server/entity/Player.java

```diff
@@ -2287,19 +2296,14 @@ public void refreshEating(@Nullable Hand eatingHand) {
      * Used to call {@link ItemUpdateStateEvent} with the proper item
      * It does check which hand to get the item to update.
      *
-     * @param allowFood true if food should be updated, false otherwise
      * @return the called {@link ItemUpdateStateEvent},
      * null if there is no item to update the state
      */
-    public @Nullable ItemUpdateStateEvent callItemUpdateStateEvent(boolean allowFood, @Nullable Hand hand) {
+    public @Nullable ItemUpdateStateEvent callItemUpdateStateEvent(@Nullable Hand hand) {
```

Comment:
Not sure about this breaking change, can we leave the method overloading this one but with a deprecation notice?